source/api_documentation/sms_messaging_api/index.rst

```diff
@@ -0,0 +1,144 @@
+====================
+SMS Messaging API
+====================
+
+.. raw:: html
+
+  <div class="alert alert-warning">
+    <b>Notice:</b>
+    The SMS Messaging API is in Alpha. Please contact your CSM or the support team for more information.
+  </div>
+
+------------------------------------
+Overview
+------------------------------------
+The SMS Messaging API provides the following functionality:
+
+- manage SMS-enabled phone numbers
+- send SMS messages
+- retrieve SMS messages and their associated Delivery Reports
+
+------------------------------------
+Authentication and Authorization
+------------------------------------
+The SMS Messaging API uses OAuth2 Tokens for authentication and authorization.
+Please see
+:doc:`../../basics/design_principles`
+for more information about generating and using an OAuth2 Token in the Invoca platform.
+
+------------------------------------
+Pagination
+------------------------------------
+The following URL parameters are used to control pagination and are common to all `GET` requests.
+
+Pagination URL Parameters
+------------------------------------
+
+.. list-table::
+  :widths: 30 50 30 30 30
+  :header-rows: 1
+  :class: parameters
+
+  * - Parameter
+    - Description
+    - Default Value
+    - Max Value
+    - Required
+
+  * - limit
+    - The maximum number of records returned per page
+    - 100
+    - 500
+    - False
+
+  * - page
+    - The next or previous page as taken directly from the response body payload of a previous API request for the previous page of the same read operation
+    - *N/A* (assumed to be the first page)
+    - *N/A*
+    - False
+
+
+------------------------------------
+Sorting
+------------------------------------
+The records returned by `GET` requests can be sorted using the `sort` URL parameter.
+The value of the `sort` parameter is comprised of:
+
+- an optional valid sortable field
+- an optional sort direction (`asc` or `desc`) separated from the sortable field by a colon (`:`)
+
+For example: `https://example.com?sort=created_at:asc`
+
+If only a sort direction is provided, the response body will be sorted in the provided direction according to the default sortable field.
+
+If only a sortable field is provided, the default sort direction (`asc`) will be assumed.
+
+Each `GET` request denotes the valid sortable fields in its response body. Please see the API examples below for more information.
+
+At this time, responses may be sorted by *one field only*.
+
+**By default, responses are sorted in ascending order according to their created_at field.**
+
+------------------------------------
+Filtering
+------------------------------------
+The records returned by `GET` requests can be filtered using URL parameters.
+Filters are comprised of:
+
+- a valid filterable field
+- a bracketed operator appended to the filterable field
+- a filter value separated from the filterable field and its associated operator by an equal sign (`=`)
```

Comment:
@jbuehring, per [this thread's discussion](https://invoca.slack.com/archives/C042LLXBSHF/p1666908097415999?thread_ts=1666905760.149919&cid=C042LLXBSHF) (and pending further discussion), we may want to remove the above.

Reply:
Going to modify to indicate the operator is required, per the changes in WEB-6455.